libconsensus/raft/RaftEngine.cpp

```diff
@@ -264,6 +265,7 @@ void RaftEngine::workLoop()
         {
             RAFTENGINE_LOG(DEBUG) << LOG_DESC("[#workLoop]Config error or I'm not a miner");
             std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
sleep 1s ? if it is too long?

Reply:
The decision that make it sleep for 1 sec is just base on feeling, if it's necessary the time of delay can be  shorter, thx.